gbpservice/nfp/service_plugins/loadbalancer/drivers/nfp_lbaasv2_plugin_driver.py

```diff
@@ -0,0 +1,28 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
```

Comment:
General Comments:
(1) Check PEP8 issues on the latest code after re-basing.
(2) Run existing unit tests of orchestrator, heat driver, configu orchestrator, configurator, FW driver and any other changed files. They should not break because of this check-in.
(3) Is there a plan to add unit tests for LBaaSV2?
(4) We need to identify devstack installation changes and multi node package installation changes for this feature.

(1), (2) and (4) are mandatory things for the feature release.